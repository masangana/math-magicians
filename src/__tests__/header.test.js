import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../components/Nav';
import { render } from '@testing-library/react';

const MockNav = () => (
  <BrowserRouter>
    <NavBar />
  </BrowserRouter>
);

test('renders correctly', () => {
  const tree = renderer
    .create(<MockNav />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
