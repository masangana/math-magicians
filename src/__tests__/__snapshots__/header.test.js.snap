// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <h3>
    Math Magicians
  </h3>
  <ul
    className="nav-links"
  >
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <h2>
      |
    </h2>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <h2>
      |
    </h2>
    <li>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
